plugins/postcss-cascade-layers/src/specificity.ts

```diff
@@ -0,0 +1,111 @@
+import selectorParser from 'postcss-selector-parser';
```

Comment:
@romainmenke Thanks for all your work on this!! I noticed this `selectorSpecificity` method seems to be used in other plugins as well, would it make sense for this to be provided centrally from postcss? Also, this code was a bit difficult to follow. Would you be able to clarify what `a`, `b`, and `c` specifically mean in relation to the specificity? I wasn't sure for example how `a` is associated with `id` and `c` with `tag`. Thanks again!

Reply:
@romainmenke thanks for the helpful context and yes absolutely, this can be done at a later time, and thanks again! I appreciate your help!

Reply:
@romainmenke that is so awesome! Thank you for taking the time to do that and in general, for always being so open to feedback! 🙏🏼 💯